admin_requests.go

```diff
@@ -73,14 +73,14 @@ func (c *Client) UserFunctions(userID string, namespace string) ([]*directive.Ru
 	return userFuncs.Functions, nil
 }
 
-// FunctionExecResults returns the 5 most recent successful execution results for the provided runnable.
+// FunctionExecResults returns the 5 most recent execution results for the provided runnable.
 func (c *Client) FunctionExecResults(runnable *directive.Runnable) (*ExecResultsResponse, error) {
 	if runnable == nil {
 		return nil, errors.New("Runnable cannot be nil")
 	}
 
 	req, err := c.adminRequestBuilder(http.MethodGet,
-		path.Join("/api/v1/results", runnable.FQFNURI), nil)
+		path.Join("/api/v2/results/by-fqfn", runnable.FQFNURI), nil)
```

Comment:
Do compute clients need to be able to support both v1 and v2? Typically I would say yes but since we are still in beta I'll leave that to your discretion.

Reply:
For now, we're not worried about it, but we'll definitely care post 1.0 👍

Reply:
so there is the very real possibility that someone will run an application in  a mixed version environment.